Assets/AppCenter/Plugins/AppCenterSDK/Auth/Shared/UserInformation.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AppCenter.Unity.Auth.Internal;
+
+namespace Microsoft.AppCenter.Unity.Auth
+{
+#if UNITY_IOS || UNITY_ANDROID
+    using RawType = System.IntPtr;
+#else
+    using RawType = System.Type;
+#endif
```

Comment:
Why do you need this here?

Reply:
Ohhh I was copying from the Auth module and didn't notice there's no RawType used here... I'll remove it.